doc/whats_new/v0.21.rst

```diff
@@ -107,6 +107,13 @@ Support for Python 3.4 and below has been officially dropped.
   when called before fit :issue:`12279` by :user:`Krishna Sangeeth
   <whiletruelearn>`.
 
+:mod:`sklearn.metrics`
+......................
+
+- |API| Corrected spelling of Harabasz score. :issue:`8657` by
```

Comment:
Clearly mention that XXX is renamed to XXX (because XXX), see e.g.,
``` 
The covariance.graph_lasso, covariance.GraphLasso and covariance.GraphLassoCV have been renamed to covariance.graphical_lasso, covariance.GraphicalLasso and covariance.GraphicalLassoCV respectively and will be removed in version 0.22. #9993 by Artiem Krinitsyn
```